zh/restore-data-using-tidb-lightning.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 使用 TiDB Lightning 恢复 Kubernetes 上的集群数据
 summary: 使用 TiDB Lightning 快速恢复 Kubernetes 上的 TiDB 集群数据。
-aliases: ['/docs-cn/tidb-in-kubernetes/stable/restore-data-using-tidb-lightning/','/docs-cn/tidb-in-kubernetes/v1.1/restore-data-using-tidb-lightning/']
+aliases: ['/docs-cn/tidb-in-kubernetes/stable/restore-data-using-tidb-lightning/','/docs-cn/tidb-in-kubernetes/v1.1/restore-data-using-tidb-lightning/','/docs/stable/tidb-in-kubernetes/reference/tools/tkctl/']
```

Comment:
This aliase is duplicate with that in `en/use-tkctl.md`

Reply:
```suggestion
aliases: ['/docs-cn/tidb-in-kubernetes/stable/restore-data-using-tidb-lightning/','/docs-cn/tidb-in-kubernetes/v1.1/restore-data-using-tidb-lightning/','/docs-cn/stable/tidb-in-kubernetes/maintain/lightning/']
```